tensorly/decomposition/tests/test_parafac2.py

```diff
@@ -13,6 +13,7 @@
     parafac2,
```

Comment:
In changes on main, we changed the imports within the test to importing tensorly as a whole, rather than the backend as T. There are a few minor adjustments below as a result.